config/environment.php

```diff
@@ -1,13 +1,18 @@
 <?php
+use fixate as f8;
+
 /*------------------------------------*\
 	USER ENVIRONMENT
 \*------------------------------------*/
 $ga_uacode = $pages->get('/settings')->ga_uacode ? $pages->get('/settings')->ga_uacode : false;
 
 $environment = array(
-	// 'env' => 'production',
+	'env' => 'production',
```

Comment:
@dewald-laubscher watch for leaving site in production mode - will confuse people downloading the boilerplate